addons/units/uniforms/combat_rolled_plaid.hpp

```diff
@@ -60,7 +60,17 @@ class CLASS(Uniform_Combat_RS_CDBS_GP_TB): CLASS(Uniform_Base) {
         uniformClass = QCLASS(Unit_Combat_RS_CDBS_GP_TB);
     };
 };
-class CLASS(Uniform_Combat_RS_CLBS_GP_BB): CLASS(Uniform_Base) {
+class CLASS(Uniform_Combat_RS_CDBS_BP_BB): CLASS(Uniform_Base) { // CDB = Plaid Dark Blue
+    scope = 2;
+    author = "GilleeDoo";
+    displayName = CSTRING(Uniform_Combat_RS_CDBS_BP_BB);
+    picture = QPATHTOF(UI\uniform_combat_rs_cdbs_bp_ca.paa);
```

Comment:
```suggestion
    picture = QPATHTOF(UI\uniform_combat_rs_cdbs_bp_ca.paa);
```

Neither does this one, although normally we only do the ui image for the LS variant and use it on both as people don't pay attention to them anyway.